KsApi/combine/Signal+Combine.swift

```diff
@@ -0,0 +1,18 @@
+import Combine
+import Foundation
+import ReactiveSwift
+
+extension Signal where Error == Never {
```

Comment:
This was an experiment in bridging code that I'm not using any more. Kind of nifty though.